packages/dendron-cli/src/commands/notes.ts

```diff
@@ -344,27 +364,65 @@ export class NoteCLICommand extends CLICommand<CommandOpts, CommandOutput> {
             } as NoteCommandData,
           };
         }
+        case NoteCommands.WRITE: {
+          const fname = checkFname(opts);
+          const vault = checkVault(opts);
+          const notes = await engine.findNotes({ fname, vault });
+          let note: NoteProps;
+
+          // If note doesn't exist, create new note
+          if (notes.length === 0) {
+            note = NoteUtils.create({ fname, vault, body });
+          } else {
+            // If note exists, update note body
+            const newBody = body || "";
+            note = { ...notes[0], body: newBody };
+          }
+          const resp = await engine.writeNote(note);
+          if (resp.error) {
+            return {
+              error: ErrorFactory.createInvalidStateError({
+                message: `write failed: ${resp.error.message}`,
+              }),
+              data: undefined,
+            };
+          } else {
+            this.print(`wrote ${note.fname}`);
+            return { data: { payload: note.fname, rawData: resp } };
```

Comment:
can we return a status that indicates whether we created a new note or aupdated an existing note?

Reply:
sure